src/motion/gopt_f77_methods.F

```diff
@@ -204,6 +218,14 @@ RECURSIVE SUBROUTINE cp_eval_at(gopt_env, x, f, gradient, master, &
                CPASSERT((SIZE(gradient) == nsize + 6))
                CALL pack_subsys_particles(subsys=subsys, f=gradient(1:nsize), fscale=-1.0_dp)
                CALL apply_cell_change(gopt_env, cell, gradient, update_forces=.TRUE.)
+               IF (keep_sg) THEN
+                  CALL spgr_apply_rotations_force(spgr, gradient)
+                  CALL spgr_apply_rotations_stress(spgr, cell, av_ptens)
+                  IF (para_env%ionode) THEN
+                     WRITE (6, *) master, para_env%mepos
```

Comment:
Left over debug code?

Reply:
Yes, I am afraid. Or rather prints to make sure I understand the code correctly.